lldb/source/Symbol/SymbolContext.cpp

```diff
@@ -95,7 +97,8 @@ bool SymbolContext::DumpStopContext(Stream *s, ExecutionContextScope *exe_scope,
       if (!name)
         name = function->GetName();
       if (name)
-        name.Dump(s);
+        pattern ? Address::DumpName(s, name.GetStringRef(), pattern)
+                : name.Dump(s);
```

Comment:
Could you just call Address::DumpName all the time here? If `pattern` is `nullptr` it will take the fast path and just dump the string won't it?

Reply:
But cool use of the `?` regardless :)